### DISCUSSION TITLE
Missing feature of the original game

### DISCUSSION THREAD

**Discussion by zenseii** [ihhub/fheroes2#7732] [Ideas]
I believe we're missing a key feature of the original game, @ihhub, @oleg-derevenetz, @Branikolog, @idshibanov, @Districh-ru and @LeHerosInconnu.

Original game:
![Screenshot_20230910_023858_Twitch](https://github.com/ihhub/fheroes2/assets/12501091/42c1081a-0c2c-40fb-a702-9fb5f62ca96f)

Don't ask me how this happened. I have no idea...

**Comment by Districh-ru** [comment]
A land ship. :)
A hero summoned a ship, believing that there was a water mill nearby... but made a mistake.

**Comment by oleg-derevenetz** [comment]
Maybe this boat is placed here by the map editor?

**Reply by zenseii** [reply]
It is the final Archibald campaign map. Hopefully NWC QA'd enough to find something like that 😅

**Comment by LeHerosInconnu** [comment]
Hello everyone,

This was the intermediate stage between the sailboat and the steamboat.
The idea was to drive the wings using a steam system that propelled the surrounding air into the boat's sail.
But this didn't work very well, as the natural wind often competed with the airflow generated by the system.
It later transpired that it worked better if the steam system powered a scaled-down version of the wings, which were positioned directly in the water.
The propulsion system, then called "Ailes-Hisse" (the inventor was French), derived from the contraction of the windmill's wings ("Ailes" in French) and the fact of hoisting the sail ("Hisser" in French), was then renamed "Hélice".

Now, why does this appear in Heroes 2?
I have absolutely no idea. :)

**Comment by ihhub** [comment]
Thats a long forgotten feature! Flying ships! We should add them...
